src/reducers/transactionReducer.js

```diff
@@ -174,28 +170,16 @@ function updateFee(state, action) {
   const feeString = action.value;
   const feeSats = bitcoinsToSatoshis(feeString);
   const feeError = validateFee(feeSats, state.inputsTotalSats);
-  const feeRate =
```

Comment:
I think removing this is going to help with some other bugs. Setting the fee for a given rate shouldn't also update the fee rate. This definitely isn't necessary in the new autoSpend flow, but I'm not 100% sure it doesn't break assumptions elsewhere.